features/raadpleeg-verblijfplaats-met-periode/in-onderzoek.feature

```diff
@@ -0,0 +1,622 @@
+      | gemeente van inschrijving (09.10) | datum aanvang adreshouding (10.30) |
+      | 0800                              | 20231014                           |
+      En de persoon is vervolgens ingeschreven op adres 'A2' met de volgende gegevens
+      | gemeente van inschrijving (09.10) | datum aanvang adreshouding (10.30) | aanduiding in onderzoek (83.10) | datum ingang onderzoek (83.20) |
+      | 0800                              | 20240516                           | <aanduiding onderzoek>          | 20240526                       |
+      Als gba verblijfplaatshistorie wordt gezocht met de volgende parameters
+      | naam                | waarde              |
+      | type                | RaadpleegMetPeriode |
+      | burgerservicenummer | 000000012           |
+      | datumVan            | 2023-07-01          |
+      | datumTot            | 2024-01-01          |
+      Dan heeft de response verblijfplaatsen met de volgende gegevens
+      | type  | datumVan.type | datumVan.datum | datumVan.langFormaat | datumTot.type | datumTot.datum | datumTot.langFormaat | gemeenteVanInschrijving.code | gemeenteVanInschrijving.omschrijving |
+      | Adres | Datum         | 2023-10-14     | 14 oktober 2023      | Datum         | 2024-05-16     | 16 mei 2024          | 0800                         | Hoogeloon, Hapert en Casteren        |
+      En heeft de verblijfplaats de volgende 'inOnderzoek' gegevens
+      | naam                             | waarde      |
+      | datumTot                         | true        |
+      | datumIngangOnderzoek.type        | Datum       |
+      | datumIngangOnderzoek.datum       | 2024-05-16  |
+      | datumIngangOnderzoek.langFormaat | 16 mei 2024 |
+      En heeft de verblijfplaats de volgende 'verblijfadres' gegevens
+      | naam                                         | waarde      |
+      | korteStraatnaam                              | Testpad     |
+      | huisnummer                                   | 8           |
+      En heeft de verblijfplaats de volgende 'adressering' gegevens
+      | naam                                         | waarde                        |
+      | adresregel1                                  | Testpad 8                     |
+      | adresregel2                                  | HOOGELOON, HAPERT EN CASTEREN |
+
+      Voorbeelden:
+      | omschrijving                                  | aanduiding onderzoek |
+      | hele categorie verblijfplaats                 | 80000                |
+      | historische categorie verblijfplaats          | 58000                |
+      | hele groep adreshouding                       | 81000                |
+      | historische groep adreshouding                | 581000               |
+      | datum aanvang adreshouding                    | 81030                |
+      | historische datum aanvang adreshouding        | 581030               |
+      | hele groep verblijf buitenland                | 81300                |
```

Comment:
Functioneel is dit toch niet correct? Deze aanduidingen kunnen toch alleen voorkomen bij een verblijfplaats buitenland? Als deze aanname klopt, hoeven de scenarios niet te worden toegevoegd. Dat heb ik al gedaan bij het verplaatsen naar de informatie service repo